src/main/webapp/app/core/course/manage/quick-actions/quick-actions.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="card mb-3" style="max-width: 35rem; width: 100%">
-    <h2 class="card-header course-quick-actions-title mb-3" jhiTranslate="entity.quickActions"></h2>
+<div class="card mb-3" style="max-width: 35rem; width: 100%; min-height: 91%">
```

Comment:
If this is the exact same card as the control center, just with a different title, we should use a wrapper component rather than copying the styles in my opinion. What do you think?